lib/mayaUsd/render/mayaToHydra/renderOverride.cpp

```diff
@@ -354,7 +363,7 @@ MStatus MtohRenderOverride::Render(const MHWRender::MDrawContext& drawContext) {
     // }
     TF_DEBUG(HDMAYA_RENDEROVERRIDE_RENDER)
         .Msg("MtohRenderOverride::Render()\n");
-    auto renderFrame = [&]() {
+    auto renderFrame = [&](bool markTime = false) {
         const auto originX = 0;
```

Comment:
In what situations is renderFrame(false) called?

Reply:
I have trouble following the logic, but there's a case below the first renderFrame where Storm is called again (looks like to draw wireframe).

Reply:
Ah, missed that... makes sense then!